ui/desktop/app/routes/scopes/scope/projects/targets/index.js

```diff
@@ -52,6 +57,23 @@ export default class ScopesScopeProjectsTargetsIndexRoute extends Route {
       this.can.can('connect target', target),
     );
 
-    return { targets: targets, totalItems };
+    let targetsWithSessions = [];
+    for (const target of targets) {
+      let sessions = await this.store.query('session', {
+        query: {
+          filters: {
+            target_id: { equals: target.id },
+            status: [
+              { equals: STATUS_SESSION_ACTIVE },
+              { equals: STATUS_SESSION_PENDING },
+            ],
+          },
+        },
+      });
+      sessions = orderBy(sessions.toArray(), 'created_time', 'desc');
```

Comment:
Did you check if this `toArray` triggers a deprecation warning?

Reply:
currently not seeing any deprecation warning being triggered, i also tried flipping [EXTEND_PROTOTYPES](https://rfcs.emberjs.com/id/0848-deprecate-array-prototype-extensions/#:~:text=For%20existing%20apps%2C%20a%20deprecation%20message%20will%20be%20displayed%20when%20EmberENV.EXTEND_PROTOTYPES%20flag%20is%20not%20set%20to%20false.) to true and no warnings related to `toArray`. 
![image](https://github.com/hashicorp/boundary-ui/assets/107949262/65184a4f-4bc7-4ec7-9097-5871e757ea4b)

Reply:
Did you try removing this from the list [here](https://github.com/hashicorp/boundary-ui/blob/66eca93f56abb8e910e3406b890cfd3f7aa11805/addons/core/addon/initializers/deprecations.js#L10)?

Reply:
thx for linking that, just tried removing it, and no new warning.